contrib/ci-builders/tekton/tekton-labs/tasks/zcash-build.yml

```
@@ -0,0 +1,51 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: zcash-build
+spec:    
+  params:
+    - name: JOBS
+      default: "4"
+      type: string
+    - name: BUILD_FOR_HOST
+      default: x86_64-pc-linux-gnu
+      type: string
+    - name: CONFIGURE_FLAGS
+      default: ""
+      type: string
+    - name: BUILD_CONTAINER
+      default: electriccoinco/zcashd-worker-ubuntu2004v2
+      type: string
+  resources:
+    inputs:
+      - name: source
+        type: git
+  steps:
+    - name: setup-venv
+      image: $(params.BUILD_CONTAINER)
+      script: |
+        #!/bin/bash
+        set -e -o pipefail
+        python3 -m venv /venv/zcash-build-venv
+        source /venv/zcash-build-venv/bin/activate
+        python -m pip install pyblake2 pyflakes python-qpid-proton pyutil pyzmq requests simplejson
+        
+    - name: build
+      image: $(params.BUILD_CONTAINER)
+      script: |
+        #!/bin/bash
+        set -o pipefail
+        cd workspace/source
+        echo $(git rev-parse HEAD)
+        echo $(git rev-parse --short HEAD)
+        if [ "$(params.BUILD_FOR_HOST)" == "x86_64-apple-darwin11" ]
```

```suggestion
        if [ "$(params.BUILD_FOR_HOST)" == "x86_64-apple-darwin18" ]
```
or preferably (both here and in other files) just check that `BUILD_FOR_HOST` begins with `x86_64-apple-darwin`.